devnets/specs/centauri.nix

```diff
@@ -107,7 +107,7 @@
               execCommands = [ "relay" "--config" configPathContainer ];
               inherit configPathSource configPathContainer pkgs packages
                 devnetTools;
-              dependsOn = dependsOnCreateClient;
+              dependsOn = dependsOnCreateConnection;
```

Comment:
@obsessed-cake shouldn’t it actually be on create channels?

Reply:
Imho there should not be dependency on runs at all. there should be just errors and restarts.
But because we have silent corruption in runs, we need dependnecy.

So if channels should be used instead of connection depends on if channels command also relays packets for channels,  then should depend on channels.  Will try to run and see if it connects.

Reply:
the dependency is because we needed to override the config, which has fields that are unknown.